app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/Message.java

```diff
@@ -43,40 +43,19 @@ public T getContent() {
         return mContent;
     }
 
-    public void setContent(T content) {
```

Comment:
None of them are used, and that's the way it should be. We wouldn't want anyone to change the type/content, id, sender id or creation date  of an existing message.